code/datums/mood_events/generic_positive_events.dm

```diff
@@ -162,7 +162,7 @@
 	timeout = 5 MINUTES
 
 /datum/mood_event/pet_borg
-	description = "<span class='nicegreen'>I just love my robotical friends!</span>\n"
+	description = "<span class='nicegreen'>I just love my robot friends!</span>\n"
```

Comment:
silicon?

Reply:
Robotic friends?

Reply:
I think this was intended to be a joke

Reply:
> 
> 
> I think this was intended to be a joke

really? It seemed that it was just a grammatical error since it said "robotical friends", I thought it would sound better if it was "I just love my robot friends.

Reply:
yeah i think robotic friends would fit better tbh

Reply:
> 
> 
> yeah i think robotic friends would fit better tbh

gotcha I fixed it now

Reply:
I was wondering since I was new, I edited the code on github instead on gitkraken, and I was wondering if you can see the changes.